docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -255,9 +255,10 @@ To use GPU acceleration on Linux, several environment variables are required or
 
       .. code-block:: bash
 
-         # configures OneAPI environment variables
+         # Required step. Configure OneAPI environment variables
          source /opt/intel/oneapi/setvars.sh
 
+         # Recommended Environment Variables
          export USE_XETLA=OFF
```

Comment:
need to remove this?

Reply:
I guess we still need it here. `export USE_XETLA=OFF` is only not necessary for Max GPU.